src/main/resources/avro/alleleAnnotations.avdl

```diff
@@ -110,6 +110,10 @@ record VariantAnnotationSet {
 A HGVSAnnotation record holds Human Genome Variation Society descriptions
 of the sequence change with respect to genomic, transcript and protein
 sequences. See: http://www.hgvs.org/mutnomen/recs.html.
+Descriptions should be provided at genomic level. Descriptions at coding level
```

Comment:
In the pre-ncRNA world, "coding variant" and "transcript variant" were essentially the same thing. That's no longer true: protein coding sequences are a subset of transcribed sequences. Although this distinction is routinely overlooked, it seems to me that the ga4gh interfaces should do the technically right thing by calling these "transcript variants" rather than "coding variants".  Specific suggestion:

`union { null, string } transcript = null;`
in lieu of
`union { null, string } coding = null;`

This means that transcript variant strings might contain c. or n. variants.